src/main/resources/org/jpeek/xsd/skeleton.xsd

```diff
@@ -110,6 +110,20 @@ SOFTWARE.
                     <xs:complexType>
                       <xs:sequence>
                         <xs:element name="op" minOccurs="0" maxOccurs="unbounded">
+                          <!--
```

Comment:
@dgarus Was not the intention of this issue to actually implement what you are describing in the 'todo' ???

Reply:
@fanifieiev, I read [How Micro Is Your Tasking?] and I think this can be a microtask. 
Why not?

Reply:
@dgarus OK)

Reply:
@dgarus This todo seems duplicated: you put a similar one in `OpsOf`